components/lib/calendar/Calendar.js

```diff
@@ -2286,26 +2286,36 @@ export const Calendar = React.memo(
         };
 
         const isDayDisabled = (day, month, year) => {
+            let isDisabled = false;
+
+            // first check for disabled dates
             if (props.disabledDates) {
                 if (props.disabledDates.some((d) => d.getFullYear() === year && d.getMonth() === month && d.getDate() === day)) {
-                    return true;
-                }
-            } else if (props.enabledDates) {
-                if (!props.enabledDates.some((d) => d.getFullYear() === year && d.getMonth() === month && d.getDate() === day)) {
-                    return true;
+                    isDisabled = true;
                 }
             }
 
-            if (props.disabledDays && currentView === 'date') {
+            // next if not disabled then check for disabled days
+            if (!isDisabled && props.disabledDays && currentView === 'date') {
                 let weekday = new Date(year, month, day);
                 let weekdayNumber = weekday.getDay();
 
                 if (props.disabledDays.indexOf(weekdayNumber) !== -1) {
-                    return true;
+                    isDisabled = true;
                 }
             }
 
-            return false;
+            // last check for enabled dates to force dates enabled
+            if (props.enabledDates) {
+                console.log(props.enabledDates);
+
+                // possible re-enable already disabled date
+                if (props.enabledDates.some((d) => d.getFullYear() === year && d.getMonth() === month && d.getDate() === day)) {
+                    isDisabled = !isDisabled;
+                }
```

Comment:
If ```disabledDays``` and ```disabledDates``` is empty, this could disable the day. Since the first two if branches aren't entered, this negates the initial declaration.

Reply:
So I think that was the original design enabled Dates only enables dates so if you say "enabledDates" by itself those are the ONLY available dates enabled.  But in your scenario you want to re-enable dates that have been disabled.